pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -204,23 +227,62 @@ func (nm *NodeManager) DiscoverNode(nodeID string, searchBy cm.FindVM) error {
 		for _, family := range ipFamily {
 			ips := returnIPsFromSpecificFamily(family, v.IpAddress)
 
-			for _, ip := range ips {
-				klog.V(2).Infof("Adding IP: %s", ip)
+			if addressMatchingEnabled {
```

Comment:
Can we also log the hostname value here?

Reply:
Done.